components/bl-popover-component/component.json

```diff
@@ -48,6 +48,18 @@
         }
       ],
       "handlerDescription": "This event is triggered when the user clicks on the button."
+    },
+    {
+      "name": "onMouseIn",
+      "label": "On Mouse In",
```

Comment:
![image](https://user-images.githubusercontent.com/15347646/218086998-86d3226f-655e-4b4c-b30c-49f643580d4d.png)

Not consistent with basic components